cnd/src/network/comit_ln.rs

```diff
@@ -106,10 +106,18 @@ impl ComitLN {
         id: LocalSwapId,
         create_swap_params: HanEtherereumHalightBitcoinCreateSwapParams,
     ) {
-        let digest = create_swap_params.clone().digest();
-
+        for (_, params) in self.swaps.iter() {
+            if *params == create_swap_params {
```

Comment:
Can we compare based on `digest` please?

Reply:
Sure thing!